localization:
  ros__parameters:
    target_body_id: "6"
    use_measurement_position: true
    twist_in_child_frame: true
    non_negative_vx: false
    deadband: 0.0001

    meas_noise.x: 0.005
    meas_noise.y: 0.005
    meas_noise.yaw: 0.01
    proc_noise.qa: 0.8
    proc_noise.qw: 0.6
    P0_diag: [0.1, 0.1, 0.05, 0.5, 0.5, 0.2]
